AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/test/Keyrings/AwsKms/AwsKmsHierarchicalKeyring/TestAwsKmsHierarchicalKeyring.dfy

```diff
@@ -320,6 +320,471 @@ module TestAwsKmsHierarchicalKeyring {
         == decryptionMaterialsOut.materials.plaintextDataKey;
   }
 
+  method {:test} TestSharedCacheWithSamePartitionIdAndSameLogicalKeyStoreName()
+  {
+    var branchKeyIdWest := BRANCH_KEY_ID;
+    var ttl : Types.PositiveLong := (1 * 60000) * 10;
```

Comment:
nit: When ever you put number, it is nice for future readers to indicate what the metric is.
i.e: This is Seconds, so this 166.66 hours.

Reply:
Got it. Thanks